pkg/mimir/mimir.go

```diff
@@ -132,6 +133,7 @@ func (c *Config) RegisterFlags(f *flag.FlagSet, logger log.Logger) {
 		"Use the '-modules' command line flag to get a list of available components, and to see which components are included with 'all'.")
 
 	f.BoolVar(&c.MultitenancyEnabled, "auth.multitenancy-enabled", true, "When set to true, incoming HTTP requests must specify tenant ID in HTTP X-Scope-OrgId header. When set to false, tenant ID from -auth.no-auth-tenant is used instead.")
+	f.BoolVar(&c.LogDedupEnabled, "log.dedup-enabled", false, "When set to true, selective log deduplicators will be activated.")
```

Comment:
I think "deduplication" sounds better, because we're talking about a feature and not some instances of anything

```suggestion
	f.BoolVar(&c.LogDedupEnabled, "log.dedup-enabled", false, "When set to true, selective log deduplication will be activated.")
```